com.regnosys.rosetta/src/com/regnosys/rosetta/Rosetta.xtext

```diff
@@ -410,9 +418,9 @@ RosettaSegment:
  * Expressions 
  */ 
 
-RosettaCallableCall: 
-	callable=[RosettaCallable|QualifiedName]
-	| implicitReceiver ?= 'item'
+RosettaReferenceOrFunctionCall returns RosettaExpression:
+	  {RosettaSymbolReference} symbol=[RosettaSymbol|QualifiedName] (explicitArguments?='(' (args+=RosettaCalcExpression (',' args+=RosettaCalcExpression)*)? ')')?
+	| {RosettaImplicitVariable} name=('item' | 'it')
```

Comment:
I added the option to use `it` instead of `item`... We should decide on a final name though.